src/fsharp/Fable.FCS/fsstrings.fs

```diff
@@ -0,0 +1,968 @@
+module internal SR.Resources
```

Comment:
Because It's not generated right now, it's just a resource. Perhaps it was generated before.